db/migrate/20210405030335_change_tag_type_in_installments.rb

```diff
@@ -0,0 +1,12 @@
+class ChangeTagTypeInInstallments < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def up
+    Installment.update_all(tag: nil)
```

Comment:
@NuckChorris wasn't sure in sql how to convert empty strings to null so I just did it this way.